src/include/mpir_gpu.h

```diff
@@ -40,6 +44,19 @@
         for temporary buffers. When stream workq and GPU wait kernels are
         in use, access APIs for GPU registered memory may cause deadlock.
 
+    - name        : MPIR_CVAR_GPU_REG_TMP_BUFS
```

Comment:
How about rename to `MPIR_CVAR_ENABLE_GPU_REGISTER` since it enables/disables `MPIR_gpu_register_host` regardless it is for temp buffer or not?